CactbotOverlay/CactbotOverlay.cs

```diff
@@ -247,6 +253,79 @@ public CactbotOverlay(CactbotOverlayConfig config)
       fast_update_timer_semaphore_.Release();
     }
 
+    public void EnableWatchers(bool check) {
+      this.watcher_state = check;
+    foreach (var watcher in this.watchers)
+        watcher.EnableRaisingEvents = check;
+    }
+ 
+    private void SetupFileWatcher(string url) {
+      if (url == "")
+        return;
+
+      foreach(var watcher in this.watchers)
+        watcher.Dispose();
+
+      List<string> path_list = new List<string>();
+      var overlay_full_path = url;
+      string base_path = null;
+
+      if (!url.Contains("://quisquous.github.io/cactbot/")) {
+        // Path of overlay if overlay is not remote!
+        overlay_full_path = Regex.Replace(overlay_full_path, @"file:[\\\/]+", "");
+        var overlay_path = Path.GetDirectoryName(url);
+        overlay_path = Regex.Replace(overlay_path, @"file:[\\\/]+", "");
+        base_path = Regex.Match(overlay_path, @"(.*cactbot)").Groups[1].ToString();
+
+        path_list.Add(base_path);
+
+        // Path of HTML resources if overlay is not remote!
```

Comment:
I don't understand this section.  Are you expecting things outside the `base_path` to be included?

Reply:
For this case:
```
Desktop/
  myOverlay/
    html/
      overlay.html
    css/
      style.css
      image.png
    js/
      script.js
      anotherscript.js
```
However I will delete this section and simply require the `html` to be in the root directory.